### ISSUE TITLE
Implementing Sequence to Sequence Autoencoder in Texar

### ISSUE THREAD

**Issue by nvidiaman** [asyml/texar#121]
Hello,

Is there an implementation of any sequence to sequence autoencoder implementation in texar?

Any code to get started to implement it? Can a sequence to sequence autoencoder be implemented by changing the loss function of the sequence to sequence example?

Thank you.

**Comment by ZhitingHu** [comment]
Just use the [examples/seq2seq_attn](https://github.com/asyml/texar/tree/master/examples/seq2seq_attn) code and replace target_text with source_text to do auto-encoding.

**Comment by nvidiaman** [comment]
@ZhitingHu thank you for the reply! Will cross-entropy loss work well for the sequence to sequence autoencoder, or using any other "reconstruction based" loss is the standard practice? 

Will cross entropy loss work well for reconstruction? 

thank you.

**Comment by ZhitingHu** [comment]
cross-entropy loss = log data-likelihood. so it's most widely used
